sstables/sstables_manager.hh

```diff
@@ -121,6 +121,7 @@ public:
     }
 
     virtual sstable_writer_config configure_writer(sstring origin) const;
+    const gms::feature_service& feature_service() const { return _features; }
```

Comment:
Let's add a specific method for this feature instead, returning a simple bool. We don't want to contaminate the sstable layer with feature service references.

Reply:
sure. added.